tests/arch/test.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
to hook this up; you can:

1) modify `docker/Dockerfile` to install `pacman-package-manager` and `arch-install-scripts` (better put it in its own commented section a.la `# Install Arch dependencies`)
2) modify [CI definition](https://github.com/go-debos/debos/blob/main/.github/workflows/ci.yaml#L122) to add an arch test e.g:
```
          - { name: "arch", case: "arch" }
```

Reply:
This doesn't quite work - `pacman-package-manager` is available only in `bookworm` and `sid`. The other package is present across the board.

Reply:
bump the container baseline to bookworm for testing this PR, we can cherry pick that into another PR later on.